scripts/zones/FeiYin/Zone.lua

```diff
@@ -101,7 +104,7 @@ function onEventFinish(player,csid,option)
         player:addKeyItem(LETTER_FROM_DALZAKK);
         player:messageSpecial(KEYITEM_OBTAINED,LETTER_FROM_DALZAKK);
     elseif (csid == 0x0017) then
-        player:setVar("SANDO92",3);
+        player:setVar("MissionStatus",3);

```

Comment:
The name of the variable isn't a bit generic and create a conflict if someone uses it in case you have several missions active from different expansions?